boards/common/nucleo/Makefile

```diff
@@ -0,0 +1,2 @@
+MODULE = board
```

Comment:
I think naming it `board` is a bad idea, it breaks the current conventions.
Having `common` boards implementing the `board` module will lead to errors.
(It happened with the cpu for https://github.com/RIOT-OS/RIOT/pull/8032)

I would rather see it called board_nucleo_common and depend on it by including Makefile.include/Makefile.features as we currently do for other boards common.

Reply:
Actually, in this case I do it on purpose, as the actual boards don't produce any archive, so this is the only compilation result produced by all the nucleo board folders

Reply:
They produce an empty and ignored `$(dirname)`.a file.

```
$ ls bin/nucleo-f411/nucleo-f411*
bin/nucleo-f411/nucleo-f411.a
bin/nucleo-f411/nucleo-f411:
```

And if a specific `nucleo-wtf3000` uses the common module, and still needs one specific C file, it will add `USEMODULE += nucleo-wtf3000` in its board `Makefile.include` which is not conventional for boards.